common/lib/xmodule/xmodule/modulestore/split_mongo/split_mongo_kvs.py

```diff
@@ -148,7 +148,10 @@ def has(self, key):
             if key.scope == Scope.children:
                 return False
 
-            b_type = key.block_scope_id.block_type
+            try:
+                b_type = key.block_scope_id.block_type
```

Comment:
In fact, you can use the same logic from line 120 (which looks at `self.VALID_SCOPES`). I think the existing code for `has` was being lenient and simply letting unknown scopes fall through, but it would probably be best `return False` early if the key doesn't have the right scope.